kasa/deviceconfig.py

```diff
@@ -56,13 +56,15 @@ class DeviceEncryptionType(Enum):
     Klap = "KLAP"
     Aes = "AES"
     Xor = "XOR"
+    Linkie = "Linkie"
```

Comment:
```suggestion
```
I'm not sure we need a new encryption entry as the encryption is still Xor although it's over https.